### ISSUE TITLE
Scenario bundles: order of tabs in backend counterintuitive

### ISSUE THREAD

**Issue by han-f** [OpenEnergyPlatform/oeplatform#1782]
## Description of the issue
I started writing a course on how to add information to the scenario bundles. I stumbled across the following: the scenarios themselves are only the second last heading in the backend. I assume users would work their way from the left to the right. So I would suggest to move the scenario button right after basic information. I think this makes the navigation of adding information easier to the users as they then already described their scenarios before they enter further context information. 

![grafik](https://github.com/user-attachments/assets/525dfa35-2533-4bc5-8b03-1c0a77a7df43)

## Ideas of solution

Describe possible ideas for solution and evaluate advantages and disadvantages.

## Workflow checklist

- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md)

cc @stap-m @l-emele @wingechr

**Comment by stap-m** [comment]
I agree that we should discuss the order. Especially, I think that "publication" should be the last tab.
But I also think that it makes sense to describe the general study stuff before the specific scenarios, since they hold for all scenarios. What about this order @han-f ?
1. Basic information
2. Study detail
3. Sectors and technologies
4. Scenarios
5. Models and frameworks
6. Publication

**Comment by han-f** [comment]
I would opt for a slightly different order, arguing for having the scenarios first, before filling in their details (sectors, models..)

    Basic information
    Study detail
    Scenarios
    Sectors and technologies
    Models and frameworks
    Publications

**Comment by stap-m** [comment]
> I would opt for a slightly different order, arguing for having the scenarios first, before filling in their details (sectors, models..)

I'm fine with that, too.
